app/code/Magento/InventoryInStorePickup/Model/ResourceModel/OrderPickupPoint/GetPickupPointByOrderId.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickup\Model\ResourceModel\OrderPickupPoint;
+
+use Magento\Framework\App\ResourceConnection;
+
+/**
+ * Class GetPickupPointByOrderId.
+ * Get Pickup Point identifier by order identifier.
+ */
+class GetPickupPointByOrderId
+{
+    const ORDER_ID        = 'order_id';
+    const PICKUP_POINT_ID = 'pickup_point_id';
+
+    /**
+     * @var \Magento\Framework\App\ResourceConnection
+     */
+    private $connection;
+
+    /**
+     * GetPickupPointByOrderId constructor.
+     *
+     * @param \Magento\Framework\App\ResourceConnection $connection
+     */
+    public function __construct(
+        ResourceConnection $connection
+    ) {
+        $this->connection = $connection;
+    }
+
+    /**
+     * Fetch pickup point identifier by order identifier.
+     *
+     * @param int $orderId
+     *
+     * @return string|null
+     */
+    public function execute(int $orderId):?string
```

Comment:
Why should pickup point id be a `string`? Maybe it's a point code?

Reply:
Fixed.